packages/contracts/contracts/test/CommonTest.t.sol

```diff
@@ -106,7 +106,7 @@ contract L2OutputOracle_Initializer is CommonTest {
     // Constructor arguments
     uint256 internal submissionInterval = 1800;
     uint256 internal l2BlockTime = 2;
-    uint256 internal startingBlockNumber = 1;
+    uint256 internal startingBlockNumber = 10;
```

Comment:
For `testFuzz_systemConfigDeposit_succeeds`, increased `startingBlockNumber` to prevent underflow error in below code.
```solidity
        uint256 _prevBlockNum = block.number - _blockDiff;
```